bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BigtableBufferedMutator.java

```diff
@@ -150,6 +267,26 @@ public void mutate(List<? extends Mutation> mutations) throws IOException {
   }
 
   /**
+   * Send the operations to the async executor asynchronously.  The conversion from hbase
+   * object to cloud bigtable proto and the async call both take time (microseconds worth) that
+   * could be parallelized, or at least removed from the user's thread.
+   */
+  private void offer(Mutation mutation) throws IOException {
+    if (activeMutationWorkers.get() == 0) {
+      initializeAsyncMutators();
+    }
+    long operationId;
+    try {
+      operationId = heapSizeManager.registerOperationWithHeapSize(mutation.heapSize());
```

Comment:
if we don't need "operationId" outside of try{}, move both "long operationId;" and "mutationsToBeSent.add(new MutationOperation(mutation, operationId));" into try {}?